src/components/form/FToggle.vue

```diff
@@ -0,0 +1,348 @@
+<template lang="pug">
+FField.FToggle(
+  :class="classes"
+  :style="style"
+  v-bind="{ name, hint, hideHint, hintTextColor, hintIcon }"
+)
+  label.FToggle__label(:for="name" :class="labelClasses")
+    .FToggle__wrapper
+      input.FToggle__hiddenCheckbox(
+        :id="name"
+        ref="toggleRef"
+        v-model="fieldValue"
+        :name="name"
+        type="checkbox"
+        :disabled="disabled"
+        @blur="handleBlur"
+        @change="handleChange"
+        @focus="handleFocus"
+        @keypress.enter="fieldValue = !fieldValue"
+      )
+      .FToggle__track(:class="toggleClasses")
+        .FToggle__track__thumb
+          FIcon.FToggle__toggledIcon(
+            v-if="!noCheckmarkIcon"
+            name="checkmark"
+            :color="toggledColor"
+            size="10"
+            :class="iconClasses"
+          )
+    span.FToggle__labelText(v-if="label") {{ label }}
+
+    span.FToggle__labelText(v-else)
+      slot(name="label")
+</template>
+  
+<style lang="stylus">
+.FToggle__label
+  position relative
+  display flex
+  cursor pointer
+  z-index 1
+  align-items flex-start
+  gap rem(12)
+
+  &--reverse
+    flex-direction row-reverse
+    justify-content space-between
+  
+.FToggle__wrapper
+  display flex
+  position relative
+  align-items center
+
+.FToggle__track
+  width rem(44)
+  height rem(24)
+  background-color var(--FToggle--color)
+  position relative
+  border-radius rem(12)
+  elevation('inset')
+  outline-style solid
+  outline-width rem(1)
+  outline-color var(--FToggle--border-color)
+  transition outline .1s, background-color .2s
+
+  &:hover, &:focus
+    outline-width rem(2)
+    outline-color var(--FToggle--hover-border-color)
+
+  &--toggled
+    background-color var(--FToggle--toggled-color)
+    outline-color var(--FToggle--toggled-border-color)
+    .FToggle__track__thumb
+      left 'calc(100% - %s)' % rem(22)
```

Comment:
I'm just curious on this one ahah, what is happening there with `'calc(100% - %s)'` ? 🤔 🔢

Reply:
It's necessary to use a stylus variable within a `calc` :smile: 
`%s` will be `rem(22)` here

Here's the doc ! https://stylus-lang.com/docs/operators.html#color-operations

Reply:
Really cool, I didn't know that ! Thanks for the docs ! 😄